source/localise-govuk-frontend/index.html.md.erb

```diff
@@ -0,0 +1,137 @@
+
+- how numbers are formatted, for example which character will be used for the thousands separator
+- the pluralisation rules used
+
+The component will determine its locale from the [`lang` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang) on the element or from the closest parent element with a `lang` attribute. If none of the component's parent elements have a `lang` attribute, the component will use a default locale of `en`.
+
+Setting the locale for the component does not automatically change the language of the text. You'll still need to pass the required text through one of the 3 configuration options described in the previous section.
+
+### Understanding pluralisation rules
+
+Different languages have different rules for pluralisation.
+
+English only has 2 forms, singular and plural. Other languages often have more forms. For example, Welsh has 6 possible plural forms, the Romance languages (French, Spanish, Italian) have 3 and many East Asian languages (Chinese, Japanese, Malaysian) do not have any plural forms.
+
+We follow the [Unicode Common Locale Data Repository classification](https://cldr.unicode.org/index/cldr-spec/plural-rules) and categorise these forms as: `zero`, `one`, `two`, `few`, `many` and `other`.
+The forms for each language will differ depending on its pluralisation. For example,
+
+- all languages use the `other` form
+- some languages will require you to pass additional forms
+- languages with no plural form only need the form `other`
+
+You should use a professional translation service to make sure your translation is accurate and that you're using the correct pluralisation form. You can also check [the Unicode Common Locale Data Repository documentation for guidance on which of these forms your locale requires and what `few` and `many` might represent for it](https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html).
+
+When providing text with plural forms, you'll need to pass an object associating each plural category with the text for that form. If you do not provide the required plural form for the locale, the component will fall back to the "other" form and warn you in the browser's console.
+
+If you're looking to keep English messages and override only one of the plural forms (`one` or `other`), you can only provide that specific form and it will be merged with our defaults.
+
+Our components will replace the `%{count}` placeholder with the number used for picking the plural category.
+
+The following example shows how to provide pluralisation options using the different ways you can [configure components with JavaScript](/configure-components-with-javascript/):
+
+With Nunjucks
+
+```njk
+{{ govukCharacterCount({
+  charactersUnderLimitText: {
+    other: '%{count} characters to go'
+    one: 'One character to go'
+  },
+  charactersOverLimitText: {
+    one: 'One character too many'
+  }
+})
```

Comment:
Couple of syntax errors:

```suggestion
{{ govukCharacterCount({
  charactersUnderLimitText: {
    other: '%{count} characters to go',
    one: 'One character to go'
  },
  charactersOverLimitText: {
    one: 'One character too many'
  }
}) }}
```

Reply:
I was seeing these examples more illustrative of the localisation options than copy-pasteable to get a CharacterCount up and running (which I'd expect more on the Character Count page). Looking at the docs and the code, we're also missing the `label` (which won't be graceful if missing either) and the `name`: 

```suggestion
{{ govukCharacterCount({
  id: 'details',
  name: 'details',
  label: {
    html: 'Can you provide more details?'
  },
  maxlength: 1000,
  charactersUnderLimitText: {
    other: '%{count} characters to go',
    one: 'One character to go'
  },
  charactersOverLimitText: {
    one: 'One character too many'
  }
}) }}
```